scripts/run-mod-settings.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+set -e
+U=http://localhost:9130
+T=testlib15
```

Comment:
Can we make this more explicit by calling it `$tenant`?

Reply:
Well. We have also $U

Reply:
Yes. I would have called that `$url` or even `$okapi`. No big, though.